jsonnet/jsonnetfile.lock.json

```diff
@@ -28,7 +28,7 @@
                     "subdir": ""
                 }
             },
-            "version": "3a6af8f5e9b16441308bcd57ad3f5bcaa03d1d68"
+            "version": "193d4934f85c9ff596d1f3e4dce7bd2da62a4d5e"
```

Comment:
Do we need to update all of these in this PR?

Reply:
This is a result fo running `jb update`, but you're right, I will remove unnecessary changes.

Reply:
fixed